examples/transactions/examples/trace_transaction.rs

```diff
@@ -15,20 +15,23 @@ use eyre::Result;
 #[tokio::main]
 async fn main() -> Result<()> {
     // Spin up a forked Anvil node.
-    // Ensure `anvil` is available in $PATH
+    // Ensure `anvil` is available in $PATH.
     let anvil = Anvil::new().fork("https://eth.merkle.io").try_spawn()?;
 
-    let url = anvil.endpoint().parse().unwrap();
-    let provider = HttpProvider::<Ethereum>::new_http(url);
-    let hash = fixed_bytes!("97a02abf405d36939e5b232a5d4ef5206980c5a6661845436058f30600c52df7"); // Hash of the tx we want to trace
+    // Create a provider.
+    let rpc_url = anvil.endpoint().parse()?;
+    let provider = HttpProvider::<Ethereum>::new_http(rpc_url);
+
+    // Hash of the tx we want to trace
+    let hash = fixed_bytes!("97a02abf405d36939e5b232a5d4ef5206980c5a6661845436058f30600c52df7");
 
     // Default tracing
     let default_options = GethDebugTracingOptions::default();
-    let res = provider.debug_trace_transaction(hash, default_options).await?;
+    let result = provider.debug_trace_transaction(hash, default_options).await?;
 
-    println!("DEFAULT_TRACE: {:?}", res);
+    println!("DEFAULT_TRACE: {:?}", result);
```

Comment:
nit: inline into format string, can run `cargo clippy --fix -- --warn clippy::uninlined_format_args`

Reply:
Fixed 👍 

Added the lint rules from Reth to apply this consistently going forward